mars/services/task/supervisor/graph_visualizer.py

```diff
@@ -147,3 +155,197 @@ def _export_subtask_to_dot(
         sio.write(f'label="{subtask.subtask_id}";\n}}')
         sio.write("\n")
         return sio.getvalue()
+
+
+def collect_verify(f):
```

Comment:
The word `verify` is a little bit confusing. How about renaming it to `collect_on_demand`?